part2/stages/Functions/install-main

```diff
@@ -30,6 +30,7 @@ create_lv() {
     fi
 
     lvcreate ${dfl_opts} --name ${name} ${opts} ${vg}
+    udevadm settle
```

Comment:
Very nice. My only question is should we be concerned with trying to limit the settle to the creation of the lv by adding the `--exit-if-exists` flag to exit once the lv node is created? Basically, in testing did you ever find yourselves hitting the 180 second timeout?

Reply:
It can take up to 3-4 seconds to calm down before it settles. I will try with the flag you mentioned and determine if it has any impact. Thanks.